components/event-sources/config/500-sources-controller.yaml

```diff
@@ -48,4 +48,4 @@ spec:
           - name: METRICS_DOMAIN
             value: kyma-project.io/event-sources
           - name: HTTP_ADAPTER_IMAGE
-            value: github.com/kyma-project/kyma/components/event-sources/cmd/httpsource-adapter
+            value: github.com/kyma-project/kyma/components/event-sources/cmd/http-adapter
```

Comment:
to make it clearer that these parts will be replaces by ko we should use strict references
means: `ko://github.com/kyma-project/kyma/components/event-sources/cmd/http-adapter`.

Reply:
Didn't know that trick, thanks! (and done)